File: show_counts.php
<!DOCTYPE html>
<html>
<head>
<title>Table Counts</title>
<style>
table {
  font-family: arial, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #dddddd;
  text-align: left;
  padding: 8px;
}

tr:nth-child(even) {
  background-color: #dddddd;
}
</style>
</head>
<body>

<h2>Table Counts</h2>

<table>
  <tr>
    <th>Table</th>
    <th>Count</th>
  </tr>
  <?php
  $file = fopen("table_counts.txt", "r");
  if ($file) {
      while (($line = fgets($file)) !== false) {
          $parts = explode(": ", trim($line));
          if (count($parts) == 2) {
              echo "<tr>";
              echo "<td>" . htmlspecialchars($parts[0]) . "</td>";
              echo "<td>" . htmlspecialchars($parts[1]) . "</td>";
              echo "</tr>";
          }
      }
      fclose($file);
  }
  ?>
</table>

</body>
</html>

File: show_counts.py
import tkinter as tk
from tkinter import ttk

def main():
    window = tk.Tk()
    window.title("Table Counts")

    tree = ttk.Treeview(window, columns=("Table", "Count"), show="headings")
    tree.heading("Table", text="Table")
    tree.heading("Count", text="Count")
    tree.pack(fill="both", expand=True)

    with open("table_counts.txt", "r") as f:
        for line in f:
            parts = line.strip().split(": ")
            if len(parts) == 2:
                table, count = parts
                tree.insert("", "end", values=(table, count))

    window.mainloop()

if __name__ == "__main__":
    main()
